coriolis/worker/rpc/server.py

```diff
@@ -96,7 +96,7 @@ def cancel_task(self, ctxt, task_id, process_id, force):
                     process_id, task_id))
             LOG.error(msg)
             self._rpc_conductor_client.confirm_task_cancellation(
-                ctxt, task_id, cancellation_details=msg)
+                ctxt, task_id, cancellation_details)
```

Comment:
Undefined `cancellation_details` in this method.

Reply:
Great catch, thank you!